dotCMS/src/integration-test/java/com/dotcms/enterprise/publishing/remote/bundler/FileBundlerTestUtil.java

```diff
@@ -117,14 +120,22 @@ public static File getFolderFilePath(final Folder folder, final File bundleRoot)
     public static File getCategoryPath(final Category category, final File bundleRoot)
             throws DotSecurityException, DotDataException {
 
+        final String bundleId = bundleRoot.getAbsolutePath()
+                .substring(bundleRoot.getPath().lastIndexOf("/") + 1);
+        final Bundle bundle = APILocator.getBundleAPI().getBundleById(bundleId);
+        bundle.setOperation(0);
+        final PushPublisherConfig pushPublisherConfig = new PushPublisherConfig(bundle);
+        final boolean categoryAddDirectly = pushPublisherConfig.getAssets().stream()
+                .anyMatch(asset -> asset.getType().equals(PusheableAsset.CATEGORY.getType()));
+
         final String liveWorking = hasLiveVersion(category.getIdentifier()) ? "live" : "working";
 
         final String categoryFilePath = bundleRoot.getPath()
                 + File.separator
                 + liveWorking + File.separator
                 + APILocator.getHostAPI().findSystemHost().getHostname()
                 + File.separator
-                + category.getInode() + ".category.dpc.xml";
+                + category.getInode() + (categoryAddDirectly ? ".category.xml" : ".category.dpc.xml");
```

Comment:
these should be constants

Reply:
this is a test class so I think it is ok to be hardcoding